testing/munin/APKBUILD

```diff
@@ -0,0 +1,72 @@
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: Stefan Wagner <stw@bit-strickerei.de>
+pkgname=munin
+pkgver=2.0.25
+pkgrel=0
+pkgdesc="A distributed monitoring/graphing tool"
+url="http://munin-monitoring.org/"
+arch="noarch"
+license="GPL"
+perl_modules="perl-rrd perl-net-snmp perl-log-log4perl perl-html-template
+perl-net-ssleay perl-net-server perl-date-manip perl-io-socket-inet6
+perl-file-copy-recursive perl-fcgi perl-uri"
+depends="$pkgname-node bash rrdtool perl $perl_modules"
+depends_dev="perl perl-module-build"
+makedepends="$depends_dev"
+install="$pkgname.pre-install $pkgname.post-install $pkgname.post-upgrade"
+subpackages="$pkgname-node"
+source="http://downloads.munin-monitoring.org/munin/stable/$pkgver/$pkgname-$pkgver.tar.gz
+Makefile.config munin.cron.d munin-node.initd munin.logrotate munin-node.logrotate"
+
+_builddir="$srcdir/$pkgname-$pkgver"
+build() {
+  cd "$_builddir"
+  cp ../Makefile.config .
+  make -j1 || return 1
```

Comment:
Return value of the last statement becomes return value of the function, so it’s unnecessary to add `|| return 1` on the last line of the function.